### PR TITLE
austin.intro.2.benavidez.daniel

### PR DESCRIPTION
**Pull request by DBenav27**

### CODE REVIEW

File: 02LessonTwo/in-class-project/index.html [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
+  <h3>Programs offered</h3>
+    <table>
+      <tr>
+        <th><img src="https://pbs.twimg.com/profile_images/638327600486719488/Z-1qM3qs_400x400.jpg"/></th>
+          <td>yoga</td>
+        <th><img src="http://static.wixstatic.com/media/f0df91_41c602d5d2154bfb94bf248939a18d01.jpg_256"/></th>
+          <td>cycling</td>
+        <th><img src="https://pbs.twimg.com/profile_images/595624096861331457/cYqHxMGn.jpg"/></th>
+          <td>crossfit</td>
+      </tr>
```
Comment (fasteddiegarcia) [inline 02LessonTwo/in-class-project/index.html:23]: Good work on this code. I'd like to see at least 4 sentences where the paragraph is and please include the contact information described in the directions. Overall, great effort!

File: 02LessonTwo/homework/index.html [hunk 2]
```diff
@@ -1,10 +1,40 @@
...
+      <p>South Park</p>
+      <img src="http://orig11.deviantart.net/ed20/f/2014/040/c/5/breaking_bad_season_4___flurry_icon_1_by_h3llb3rg-d75q0h2.png"/>
+      <p>Breaking Bad</p>
+  <h2>Some of my Favorite Websites</h2>
+    <ul>
+      <li><a href="http://www.codecademy.com">Code Academy</a></li>
+      <li><a href="http://www.espn.com">ESPN</a></li>
+    </ul>
 </body>
 </html>
```
Comment (ethanstenis) [inline 02LessonTwo/homework/index.html:40]: Great job, Daniel! Good use of the table for the TV show section.

File: 03LessonThree/in-class-project/css/daily-planet-style.css [hunk 3]
```diff
@@ -7,18 +7,55 @@ header, nav, main, section, aside, footer {
...
+}
+figcaption{
+  text-align:center;
+  padding-left:30px;
+  text-decoration:underline;
+  font-size:120%;
+}
+aside p{
+  margin-left:30px;
+}
```
Comment (ethanstenis) [inline 03LessonThree/in-class-project/css/daily-planet-style.css:61]: Great job taking the CSS beyond the provided code! Keep pushing yourself with new ideas and you will only get better.

File: 03LessonThree/in-class-project/daily-planet.html [hunk 4]
```diff
@@ -1,10 +1,48 @@
...
+        <h1>Daily Planet</h1>
+        <nav>
+          <a href="#local">Local</a>
+          <a href="#sports">Sports</a>
+          <a href="#Economy">Economy</a>
+        </nav>
+      </header>
+      <main>
+        <section>
+          <h2>Rio Olympics athletes advised to not put their heads underwater</h2>
```
Comment (ethanstenis) [inline 03LessonThree/in-class-project/daily-planet.html:19]: Your code looks good. It is missing one element, "article", which you would nest just inside the "section" element. You would then nest the "h2"... inside the "article". Add in the "article" element properly and I can bump your grade up to a 100.

File: 04LessonFour/in-class-project/css/style.css [hunk 5]
```diff
@@ -1,84 +1,21 @@
...
+  font-family:Helvetica,Arial,Sans-Serif;
 }
-
-.head {
-    width: 100%;
-    height: 200px;
-    border: solid 3px black;
+.navbar{
+  background-color:#0071c5;
+  color:white;
```
Comment (ethanstenis) [inline 04LessonFour/in-class-project/css/style.css:8]: Add some padding here to get some room around that h1.

File: 04LessonFour/in-class-project/css/style.css [hunk 6]
```diff
@@ -1,84 +1,21 @@
...
-    border: solid 2px black;
+.core {
+  background-color:#ffffff;
 }
-
-.head h1 {
-    text-align: center;
-    margin-top: 10px;
+.Customer {
+  background-color:lightgrey;
```
Comment (ethanstenis) [inline 04LessonFour/in-class-project/css/style.css:15]: Add some padding here to get some room around the h2.

File: 12LessonTwelve/in-class-project/index.html [hunk 7]
```diff
@@ -1,10 +1,45 @@
...
+           <img src="img/icon_home_4.svg" />
+             <h2>Search by address</h2>
+             <p>Search by the area</p>
+         </div>
+     </section>
+     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+
 
 </body>
```
Comment (ethanstenis) [inline 12LessonTwelve/in-class-project/index.html:44]: Way too much space between icons in the phone view. Make this change and I can bump your grade up. Everything else looks great!
screen shot 2016-09-13 at 5 57 05 pm
